api/src/main/java/org/eclipse/microprofile/metrics/package-info.java

```diff
@@ -89,11 +89,9 @@
  * <pre>
  * <code>
  *     Timer timer = metricRegistry.timer(metadata);
- *     Timer.Context context = timer.time();
- *
- *     ... // code that will be timed
- *
- *     context.close();
+ *     try (Timer.Context context = timer.time()) {
+ *         ... // code that will be timed
+ *     }
```

Comment:
will need a `context.stop()` or `context.close()` to stop the timer.

Reply:
the try-with-resources block handles the automatic closing of the context when the block completes